pkg/k8s/labels.go

```diff
@@ -334,3 +334,33 @@ func IsTapDisabled(pod *corev1.Pod) bool {
 	}
 	return false
 }
+
+// GetAnnotationsDocs return map with annotations and its descriptions
+// description can be omitted so it will be taken from corresponding cli flag
+func GetAnnotationsDocs() map[string]string {
```

Comment:
Since all the keys in the map are exported and this is only used in `cli/cmd/doc.go`, we can just move this to that file, so that it's close to where it's used.